### ISSUE TITLE
Try online

### ISSUE THREAD

**Issue by teodorov** [plug-obp/plug-obp.github.io#1]
It will be nice to have a simple try online thingy...

**Comment by jeancharles-roger** [comment]
I agree it would be nice. We need a dedicated server for that, do you have access to one ?

**Comment by teodorov** [comment]
I think about a virtualised server (hosted by us).
On the long run we might want to integrate it with supercomputers (using
slurm job manager).
The priority should be :
- petits explorations < 2-4 Go

upload fichiers -> run button -> résultats

It should be easy to create, and a good starting base.

Couldn't we have a docker config for something like that ?
We need to ask the SI what is their virtualization technology that they
prefer.

**Comment by jeancharles-roger** [comment]
I can easily create a docker image for this
